.github/workflows/PullRequestClosed.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request Closed
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  PullRequestMerged_job:
+    name: Pull Request Merged
+    runs-on: ubuntu-latest
+    environment: jira
```

Comment:
This line is different from the example. And I think I saw you removed it in some repo already. 
Should it be removed here as well? 
Same for the other files.

Reply:
There are few repos that need it, but this one is not on my let. I'll check

Reply:
It's gone, good catch!

I used "Request review" from "Sebastian Marichal - you can ignore that noise